src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsMoreEmptyValues.java

```diff
@@ -13,7 +13,7 @@
 import java.lang.reflect.Array;
 
 /**
- * It's likely this implementation will be used by default by every Mockito 2.0.0 mock.
+ * It's likely this implementation will be used by default by every Mockito 2.1.0 mock.
```

Comment:
Why not, you advocated on the google code tracker the change. I forgot about it too. I think this answer is good.

If you agree and I agree too then let's make the change ;)